boards/risc-v/bl602/bl602evb/configs/wifi/defconfig

```diff
@@ -73,6 +73,7 @@ CONFIG_MTD=y
 CONFIG_MTD_PARTITION=y
 CONFIG_NET=y
 CONFIG_NETDB_DNSCLIENT=y
+CONFIG_NETDEV_LATEINIT=y
```

Comment:
Why do we need this?

Reply:
since those architectures don't implement the "standard" up/riscv_netinitialize, but they initialize the network in board files directly with some custom interface.